ci/verify-chefdk.sh

```diff
@@ -6,7 +6,7 @@ export TMPDIR
 rm -rf $TMPDIR
 mkdir -p $TMPDIR
 
-export PATH=/opt/chefdk/bin:$PATH
+export PATH=/opt/chefdk/bin:/opt/chefdk/embedded/bin:$PATH
```

Comment:
Maybe we should only do this for acceptance testing? Generally we want to avoid relying on having embedded/bin stuff in the path.